lib/MusicBrainz/Server/Data/Place.pm

```diff
@@ -37,7 +37,7 @@ sub _type { 'place' }
 
 sub _columns
 {
-    return 'place.id, place.gid, place.name, place.type, place.address, place.area, place.coordinates[0] as coordinates_x, ' .
+    return 'place.id, place.gid, place.name COLLATE musicbrainz, place.type, place.address, place.area, place.coordinates[0] as coordinates_x, ' .
```

Comment:
Maybe too nitpicky, but this for example is like a very long line (146 chars) which probably would benefit from a split. Might be true of all lines here though...

Reply:
Agreed, but it was already well over the length limit, I just made it slightly worse. :)

I think we should change all of the `_columns` implementations to do something like `join q(, ), (...)` so we can put each item on its own line.  But then they should also probably be changed to lazy Moose attributes rather than subroutines, so that the joined value is cached.  And I thought of doing all this, but didn't really want to complicate this PR any further.

Reply:
When around, maybe you can make a ticket for your idea and we can implement it later on? :) For now we can just merge as-is